pkg/leeway/reporter.go

```diff
@@ -225,3 +228,148 @@ func (r *WerftReporter) PackageBuildFinished(pkg *Package, err error) {
 	}
 	fmt.Printf("[%s|%s] %s\n", pkg.FullName(), status, msg)
 }
+
+type PackageReport struct {
+	logs     strings.Builder
+	start    time.Time
+	duration time.Duration
+	status   PackageBuildStatus
+	err      error
+}
+
+func (r *PackageReport) StatusIcon() string {
```

Comment:
As the report is for a completed build, when would we see anything except a success or a failure? E.g. "PackageBuilding"?

Reply:
When one package fails to build, Leeway does not necessarily call `func (r *MarkdownReporter) PackageBuildFinished(pkg *Package, err error) {` on other packages. Thus, those packages remain at their current status: 
* PackageBuilding: if the package build started
* PackageNotBuiltYet: Otherwise